src/Generator/BelboonDE.php

```diff
@@ -364,7 +334,7 @@ private function getShipping($variation):string
             $shippingCost = $this->shippingCostCache[$variation['data']['item']['id']];
         }
 
-        if(!is_null($shippingCost))
+        if(!is_null($shippingCost) && $shippingCost > 0)
```

Comment:
shipping costs can be zero

Reply:
from the belboon documentation the only condition is to be a `varchar(255)`, there aren't any restriction to `empty` string or formatted string `0.00`